ibm/mas_devops/roles/suite_install/templates/core_v1_suite.yml.j2

```diff
@@ -83,6 +83,8 @@ spec:
 {% if mas_channel != '8.7.x' and mas_channel != '8.8.x' %}
     manualCertMgmt: {{  mas_manual_cert_mgmt  }}
 {% endif %}
+    userDataValidation:
```

Comment:
Wrap this in a `{% if mas_special_characters != '' %}` conditional so that we only add it when the user chooses to set this config option